### DISCUSSION TITLE
Unread Only Feature

### DISCUSSION THREAD

**Discussion by ghost** [jocmp/capyreader#274] [Feature Requests]
Can we add an unread only feature? It would be nice if there was an option to show only unread posts in a feed. Otherwise it gets very confusing. 

![Screenshot_20240816_164034_Feedly](https://github.com/user-attachments/assets/7906176e-8f6f-4ae9-afdc-76e9416c858a)

In the meantime, I entered the Turkish translation data.

**Comment by ghost** [comment]
![Screenshot_20240819_123901_Feedly](https://github.com/user-attachments/assets/50d4fdd0-764f-4ee1-836d-27faf3e50254)

It would be nice if the screen would look like this or a similar quick blank screen and cleaning feature when articles in any publication are marked as read.

**Reply by jocmp** [reply]
Agreed, I need improve empty states on read feeds. I'll think about when I can add this in.

**Comment by jocmp** [comment]
Unread only is a feature already, it's a filter in the drawer that will only show unread articles for each tag or feed group.

![Screenshot_20240820-001715.png](https://github.com/user-attachments/assets/32b9eddb-59b8-49f6-aeb7-25061f823961)

**Reply by amit3s** [reply]
Hello, I didn't see a way to only filter for unread articles. I'm running version 2024.10.1069-dev from fdroid.

**Reply by amit3s** [reply]
Ah, I see the unread filter, but it keeps articles that are read in that session around - that confused me.

**Reply by jocmp** [reply]
Hey @amit3s - yep, you're right. Unreads are cleared either when you swipe to refresh or switch to a different folder or feed.
